steps/unity-install/0.1.0/step.yml

```diff
@@ -0,0 +1,92 @@
+title: Unity Install
+summary: |
+  Downloads and installs Unity packages
+description: |
+  Downloads and installs Unity packages
+website: https://github.com/bdrelling/bitrise-step-unity-install
+source_code_url: https://github.com/bdrelling/bitrise-step-unity-install
+support_url: https://github.com/bdrelling/bitrise-step-unity-install/issues
+published_at: 2019-03-21T11:28:54.545386-05:00
+source:
+  git: https://github.com/bdrelling/bitrise-step-unity-install.git
+  commit: a168322102969eb88d649d0d3304e756da26df9c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- xamarin
+type_tags:
+- installer
```

Comment:
I think it would be a good idea to add `utility` next to the installer type_tag:

```
type_tags:
- installer
- utility
```

Reply:
Can you explain why so I understand?